bitnami/gitea/templates/service-account.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "gitea.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: gitea
```

Comment:
Is it a conscious decision to only add this label for the SA resource?

Reply:
Copy-paste :sweat_smile: